exporter/influxdbexporter/writer.go

```diff
@@ -119,10 +119,13 @@ type influxHTTPWriterBatch struct {
 func (b *influxHTTPWriterBatch) WritePoint(_ context.Context, measurement string, tags map[string]string, fields map[string]interface{}, ts time.Time, _ common.InfluxMetricValueType) error {
 	b.encoder.StartLine(measurement)
 	for _, tag := range b.sortTags(tags) {
-		b.encoder.AddTag(tag.k, tag.v)
+		if tag.v != "" {
+			val := fmt.Sprintf("%v", tag.v)
+			b.encoder.AddTag(fmt.Sprintf("t_%v", tag.k), val)
```

Comment:
what kind of values are `tag.k` and `tag.v` ?
Just curious since you are using the `%v` formatting directive which i think of as a code smell thing maybe from the fact that I use and abuse it in debug logging

Reply:
yeah, let me see if I can clean this up a bit, maybe we don't need to.

Reply:
plus sign also works (apparently), but if %s is more conventional I can change!

Reply:
We know `k` and `v` will be strings based on the type of `tag`, since `tags` is a `map[string]string`, so I opted for the `+` string concatenation